x-pack/plugins/security_solution/public/resolver/view/process_event_dot.tsx

```diff
@@ -338,9 +340,10 @@ const UnstyledProcessEventDot = React.memo(
             left: '0',
             outline: 'transparent',
             border: 'none',
+            pointerEvents: 'none',
           }}
         >
-          <g>
+          <g fill="none" style={{ pointerEvents: 'visiblePainted' }}>
```

Comment:
I feel like we are very in between using styled components and in-line styles in this file and generally in the app. This file has other changes that need to take place, but I feel like we should move entirely to using styled components as intended so we have all the CSS logic in one place. Not something you have to do here, but this reminded me of that

Reply:
that's a good point. I know we mentioned in the past there are performance/rendering complications from some styled things, but this could probably be moved to a styled component with no problems and it would likely work better that way. I'll change this before I merge it in.

Reply:
Done here https://github.com/elastic/kibana/pull/76997/commits/18598d28d605b666e981e91ad8962ac5f6b640e2